### ISSUE TITLE
Missing companyName field in Address Dto

### ISSUE THREAD

**Issue by nikgilbe** [jlevers/selling-partner-api#692]
## Problem description:
The shipping Address companyName is missing from vendor/jlevers/selling-partner-api/src/Seller/OrdersV0/Dto/Address.php. 

It is showing up in the json response, but not the dto. 
## Error:

```
Warning: Unknown keys when deserializing into SellingPartnerApi\Seller\OrdersV0\Dto\Address: CompanyName

```

## Code
Changing Address.php to this resolves it:
```php
use Crescat\SaloonSdkGenerator\BaseDto;

final class Address extends BaseDto
{
    protected static array $attributeMap = [
        'name' => 'Name',
        'companyName' => 'CompanyName',
        'addressLine1' => 'AddressLine1',
        'addressLine2' => 'AddressLine2',
        'addressLine3' => 'AddressLine3',
        'city' => 'City',
        'county' => 'County',
        'district' => 'District',
        'stateOrRegion' => 'StateOrRegion',
        'municipality' => 'Municipality',
        'postalCode' => 'PostalCode',
        'countryCode' => 'CountryCode',
        'phone' => 'Phone',
        'addressType' => 'AddressType',
    ];

    /**
     * @param  ?string  $name  The name.
     * @param  ?string  $addressLine1  The street address.
     * @param  ?string  $addressLine2  Additional street address information, if required.
     * @param  ?string  $addressLine3  Additional street address information, if required.
     * @param  ?string  $city  The city
     * @param  ?string  $county  The county.
     * @param  ?string  $district  The district.
     * @param  ?string  $stateOrRegion  The state or region.
     * @param  ?string  $municipality  The municipality.
     * @param  ?string  $postalCode  The postal code.
     * @param  ?string  $countryCode  The country code. A two-character country code, in ISO 3166-1 alpha-2 format.
     * @param  ?string  $phone  The phone number. Not returned for Fulfillment by Amazon (FBA) orders.
     * @param  ?string  $addressType  The address type of the shipping address.
     */
    public function __construct(
        public readonly ?string $name = null,
        public readonly ?string $companyName = null,
        public readonly ?string $addressLine1 = null,
        public readonly ?string $addressLine2 = null,
        public readonly ?string $addressLine3 = null,
        public readonly ?string $city = null,
        public readonly ?string $county = null,
        public readonly ?string $district = null,
        public readonly ?string $stateOrRegion = null,
        public readonly ?string $municipality = null,
        public readonly ?string $postalCode = null,
        public readonly ?string $countryCode = null,
        public readonly ?string $phone = null,
        public readonly ?string $addressType = null,
    ) {
    }
}```

**Comment by nikgilbe** [comment]
FYI, this is also the case in v5.x (vendor/jlevers/selling-partner-api/lib/Model/OrdersV0/Address.php. Similar fix worked there

**Comment by jlevers** [comment]
This will be fixed in v7, which will be released soon.

**Comment by jlevers** [comment]
Released in `v7.0.0`.
